### PR TITLE
Send File to Webhook

### PR DESCRIPTION
**Pull request by TheRealPikachu**
**Please describe the changes this PR makes and why it should be merged:**

**Status**

- [ ] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes
- [ ] Documentation has been added/modified, or there is nothing to change (docs/mods.json)

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

File: actions/send_file_to_webhook_MOD.js [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+  // so be sure to provide checks for variable existance.
+  // ---------------------------------------------------------------------
+
+  action (cache) {
+    const data = cache.actions[cache.index]
+    const url = this.evalMessage(data.url, cache)
+    const filename = this.evalMessage(data.filename, cache)
+    const Mods = this.getMods()
+
+    const { Webhook } = Mods.require('discord-webhook-node')
```
Comment (almeidx) [inline actions/send_file_to_webhook_MOD.js:104]: no

Comment (DetroitWhiskey136) [reply actions/send_file_to_webhook_MOD.js:40]: try doing this shit without using outside packages first. pretty sure discord.js already fully supports webhooks

Comment (TheRealPikachu) [reply actions/send_file_to_webhook_MOD.js:40]: ok danno

**Comment by TheRealPikachu** [comment]
yeah got it, I'll resubmit tomorrow

**Comment by TheRealPikachu** [comment]
Just testing right now

**Pull request closed without merging**